docs/Concepts/5.md

```diff
@@ -1,19 +1,11 @@
 ---
-title: Research in Progress
+title: Transports in Waku
 ---
 
-The following features are currently experimental and under research and initial implementation:
+Waku v2 is built in top of libp2p, and like libp2p it strives to be transport agnostic. We define a set of recommended transports in order to achieve a baseline of interoperability between clients.
 
-**Economic Spam resistance**:
-We aim to enable an incentivized spam protection technique to enhance `WAKU-RELAY` by using rate limiting nullifiers.
-More details on this can be found in [17/WAKU2-RLN-RELAY](https://rfc.vac.dev/spec/17).
-In this advanced method, peers are limited to a certain rate of messaging per epoch and an immediate financial penalty is enforced for spammers who break this rate.
+This section describes these recommended transports:
 
-**Prevention of Denial of Service (DoS) and Node Incentivization**:
-Denial of service signifies the case where an adversarial node exhausts another node's service capacity (e.g., by making a large number of requests) and makes it unavailable to the rest of the system.
-DoS attack is to be mitigated through the accounting model as described in [18/WAKU2-SWAP](https://rfc.vac.dev/spec/18).
-In a nutshell, peers have to pay for the service they obtain from each other.
-In addition to incentivizing the service provider, accounting also makes DoS attacks costly for malicious peers.
-The accounting model can be used in `13/WAKU2-STORE` and `12/WAKU2-FILTER` to protect against DoS attacks.
-
-Additionally, this gives node operators who provide a useful service to the network an incentive to perform that service.
+- Waku nodes uses TCP to communicate by default. A service node should be using TCP to connect to other nodes and listen to
```

Comment:
```suggestion
- Waku nodes uses TCP to communicate by default. A service node should be using TCP for communicate with other nodes.
```

Reply:
A section of state of NAT would be good here so reader know what to expect. cc @richard-ramos @jm-clius